sos/plugins/libvirt.py

```diff
@@ -69,12 +70,17 @@ def setup(self):
                 self.add_copy_spec("/proc/%s/%s" % (pid, pf))
 
     def postproc(self):
-        for loc in ["/etc/", "/var/run/"]:
-            for xmlfile in glob.glob(loc + "libvirt/qemu/*.xml"):
-                self.do_file_sub(
-                    xmlfile,
-                    r"(\s*passwd=\s*')([^']*)('.*)",
-                    r"\1******\3"
-                )
+        # get files
+        config_files = [glob.glob('/etc/libvirt/qemu/*.xml'),
+                        glob.glob('/var/run/libvirt/qemu/*.xml'),
+                        glob.glob('/etc/libvirt/*.conf')]
+
```

Comment:
I'm not keen on this: `config_files` implies a list of config files, but it's not: it's a list of lists or config  files, and then it gets flattened out in the weird looking `y for x in c for y in x`.

Why the extra complexity? Why not just assign `config_files` from the sum (append) of the three glob lists?